content/vault/v1.21.x (rc)/content/api-docs/system/internal-counters.mdx

```diff
@@ -1004,6 +1004,204 @@ $ curl \
     http://127.0.0.1:8200/v1/sys/internal/counters/activity?end_time=2020-06-30T00%3A00%3A00Z&start_time=2020-06-01T00%3A00%3A00Z&limit_namespaces=2
 ```
 
+## Cumulative namespace client count
+
+@include 'alerts/enterprise-only.mdx'
+
+<Note title="Tech preview">
+
+  Tech preview provides access to new features for experimentation and testing.
+  Feature behavior, CLI commands, and API requests/response schemas may change
+  significantly in future releases.
+
+  **We strongly discourage using tech preview features in production deployments
+  of Vault**
+
+</Note>
+
+The cumulative client count endpoint returns a by-namespace breakdown of client
+activity for the given billing period. You can call the cumulative client count
+endpoint from all namespaces and the cumulative total for each namespace includes
+client activity in said namespace plus all descendant namespaces.
+
+To query client counts for deleted namespaces, you must call the endpoint from
+`root` since the original namespace path is unknown. As a result,
+deleted namespaces do not contribute to the cumulative counts of any namespace
+other than `root`. 
+
+<Tip title="Related endpoint">
+
+For client counts isolated to each namespace, grouped by month, or to see
+mount-level attribution, use `sys/internal/counters/activity`.
+
+</Tip>
+
+Regardless of the values provided for `start_time` and `end_time`, Vault can
+only return totals for whole billing periods. If the requested time parameters
+do not align with a complete billing period, Vault adjusts the request to the
+full billing period associated with the provided times.
+
+The endpoint response always includes the actual time period covered in addition
+to the client counts. If Vault adjusts the query period, the result also
+includes a warning about the adjusted query times.
+
+If `start_time` and `end_time` do not fall within the same billing period, Vault
+uses the billing period associated with `start_time`.
+
+<Warning>
```

Comment:
@schavis I added this warning back in, it was removed but I didn't see the information included elsewhere

Reply:
I know I edited it because I moved the text and removed the warning tag 🤔 I'm not sure what happened

Reply:
It probably blended in with the other text 😄 Thanks for editing it!